website/ru.json

```diff
@@ -1,35 +1,37 @@
 {
   "global": {
-    "home": "Зашифровать файлы",
-    "about": "О нас",
-    "adventure": "Цели и достижения",
+    "home": "Шифрование файлов",
+    "about": "О Muonium",
```

Comment:
this means "about Muonium", right? But the English translations mean "About us", so the previous one is OK.

Reply:
Maybe in Russia it's more common to say "About xxx" than "About us"? I don't know

Reply:
I don't think so. The previous one who translated it put "About us", then it's not more common or idk, it's just an error from this translator.